Templates/VisualBasic/UWP/ProjectTemplates/VisualBasic/Windows UAP/UWP_VB/UnitTestAppMainPage.xaml.vb

```diff
@@ -0,0 +1,9 @@
+﻿' The Blank Page item template is documented at https://go.microsoft.com/fwlink/?LinkId=234238
```

Comment:
Remove this comment.
The C# version doesn't have an equivalent and the link is out of date.